src/main/java/use_case/ports/api/TranslateService.java

```diff
@@ -1,4 +1,4 @@
-package interface_adapter.api.google_translate;
+package use_case.ports.api;
```

Comment:
Hi! Great work on reverting the dependencies. However, I'm not sure if the TranslateService belongs in use_case, cos it feels more like a service interface that should be in entities since any API will be able to implement it and similarly to how I structured strategy interfaces.

From the CA Login example, they put their password validator service in entity as well, but since the actual implementation is an API with interface adapters, I believe it's still okay since you can the created concrete class that implements the entity service will point inwards and is also allowed to use translation commands since its outside enough. LMK what you think!

Reply:
Also do you think putting those services (ex: related words strategies, translater service, text to speech service) would be entity elements we can provide in the use cases since they technically are allowed to point inside for entities, or do they belong with the data base and file word DAO file?I think the DAO is allowed to point to strategies and services but it might be more appropriate to keep the entities inside with the interactor since it has more direct to entities. iirc, the only time data base interacts directly with entities is when its creating and saving the objects, and all other alterations should happen with our interactor, Do you agree?